### PR TITLE
kanata: add version 1.3.0

### PR DESCRIPTION
**Pull request by storvik**
Add kanata version 1.3.0 to Extras. Kananta is a software keyboard remapper written in Rust. 

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by storvik** [comment]
@HUMORCE  @r15ch13 

Just a friendly ping, hoping to get this merged soon 😄 

Kanata also ships a version that can run shell commands, but this is disabled by default due to security concerns. Would it be possible to add a package `kanata-with-cmd` that install the `kanata_cmd_allowed.exe`? This is how `nixpkgs` solved it, but not sure how Scoop usually does these things.

File: bucket/kanata.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+    "version": "1.3.0",
+    "description": "A software keyboard remapper.",
+    "homepage": "https://github.com/jtroo/kanata",
+    "license": "LGPL-3.0-only",
+    "notes": [
+        "See sample config here: https://github.com/jtroo/kanata/blob/main/cfg_samples/kanata.kbd"
```
Comment (HUMORCE) [inline bucket/kanata.json:7]:
```suggestion
        "Configuration Guide: https://github.com/jtroo/kanata/blob/main/docs/config.adoc"
```

**Comment by HUMORCE** [comment]
> Kanata also ships a version that can run shell commands, but this is disabled by default due to security concerns. Would it be possible to add a package kanata-with-cmd that install the kanata_cmd_allowed.exe? This is how nixpkgs solved it, but not sure how Scoop usually does these things.

Create manifests for other variants as well.

**Comment by storvik** [comment]
Not sure, but maybe it's the # on line 8? Is it possible to escape it with \ or should we just remove the last part from the URL?

**Comment by HUMORCE** [comment]
> Is it possible to escape it with \ or should we just remove the last part from the URL?

https://github.com/ScoopInstaller/Scoop/wiki/App-Manifest-Autoupdate#url-variables

**Comment by storvik** [comment]
Yeah, just noticed the missing commas. Sorry about that!

**Pull request merged**